### PR TITLE
:zoom-in with too large count doesn't change zoom #1118

### PR DESCRIPTION
**Pull request by axs221**
This is related to issue #1118. typing 999999999+ should result in the max standard zoom, 500%. typing 999999999- should result in min standard zoom, 25%. Minor refactorings, mainly making the __snap_in method able to run whether current zoom is fuzzy or not for consistency.

### CODE REVIEW

File: tests/integration/features/zoom.feature [hunk 1]
```diff
@@ -17,19 +17,23 @@ Feature: Zooming in and out
 
     Scenario: Zooming in with count
         When I run :zoom-in with count 2
-        Then the message "Zoom level: 120%" should be shown.
-        And the zoom should be 120%
+        Then the message "Zoom level: 125%" should be shown.
+        And the zoom should be 125%
```
Comment (The-Compiler) [inline tests/integration/features/zoom.feature]: Seeing `zoom-levels` is set to something including `120%` but not `125%`, why did this change?

File: tests/integration/features/zoom.feature [hunk 2]
```diff
@@ -17,19 +17,23 @@ Feature: Zooming in and out
...
 
-    # https://github.com/The-Compiler/qutebrowser/issues/1118
     Scenario: Zooming in with very big count
         When I run :zoom-in with count 99999999999
-        Then the message "Zoom level: 100%" should be shown.
-        And the zoom should be 100%
+        Then the message "Zoom level: 500%" should be shown.
+        And the zoom should be 500%
+
+    Scenario: Zooming in with very big count
```
Comment (The-Compiler) [inline tests/integration/features/zoom.feature]: This should be "Zooming out".

**Comment by The-Compiler** [comment]
Huh, what's the reason for closing this?

**Comment by axs221** [comment]
I realized it wasn't working as intended. I am reworking it now and I will
make sure it is working correctly before opening a request.


Shawn Axsom

**Comment by axs221** [comment]
New request has been opened. The integration tests have been corrected and
the issues I noticed should be resolved:

https://github.com/The-Compiler/qutebrowser/pull/1140



Shawn Axsom

**Comment by The-Compiler** [comment]
Note you can push new commits to an existing PR to update it :wink:

**Pull request closed without merging**